misk/src/main/kotlin/misk/client/HttpClientsConfig.kt

```diff
@@ -125,45 +85,30 @@ data class HttpClientConfig(
   val readTimeout: Duration? = null,
   val pingInterval: Duration? = null,
   val callTimeout: Duration? = null,
-  val maxRequests: Int = 128,
-  val maxRequestsPerHost: Int = 32,
-  val maxIdleConnections: Int = 100,
-  val keepAliveDuration: Duration = Duration.ofMinutes(5),
+  val maxRequests: Int? = null,
+  val maxRequestsPerHost: Int? = null,
+  val maxIdleConnections: Int? = null,
+  val keepAliveDuration: Duration? = null,
   val ssl: HttpClientSSLConfig? = null,
-  val unixSocketFile: File? = null,
+  val unixSocketFile: String? = null,
```

Comment:
Changed to String, as there's a weird Jackson issue, where it's incorrectly deserializing `\u0000` character into `File`, but correctly deserializes same exact value into `String`. Covered by test.